src/modules/songs/components/SongTagSelector/SongTagSelector.tsx

```
@@ -0,0 +1,407 @@
+            {showCreateOption ? (
+              <div className="px-3 py-3">
+                <button
+                  onClick={handleCreateTag}
+                  className={cn(
+                    "flex w-full cursor-pointer items-center gap-2 rounded-lg px-3 py-2 text-sm transition-colors",
+                    highlightedIndex === 0
+                      ? "bg-primary text-primary-foreground hover:bg-primary/90"
+                      : "hover:bg-secondary/50",
+                  )}
+                  data-index={0}
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)